src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java

```diff
@@ -532,6 +524,19 @@ private static void visitClass(DetailAST ast) {
             }
         }
 
+        private static void visitField(DetailAST ast, DetailAST node) {
+            if (ScopeUtil.isInScope(node, Scope.PUBLIC)) {
```

Comment:
I know the name came from the other method, but node is a bad name. It should have been named something like `parentAst`. I saw node and was thinking we were checking the modifier of the field, not the parent.

Reply:
`ast` named variables are not good names, even it used in 90% in our code. AST - abstract syntax tree. We have not whole tree but only some node of it.
Does it make sense to name `node` and `parentNode` ?

Reply:
I fine as long as we change it signify the parent in some way.

Reply:
Done. I'm not going to to rename all variables - it's a separate issue, but this method and its siblings changed.